components/layout/navbar/NavbarRoot.tsx

```diff
@@ -104,7 +104,9 @@ function NavListRow({
 }) {
   return (
     <li
-      className={`text-white font-yoon text-md font-medium ${highlight ? '' : 'opacity-60'}`}
+      className={`text-white font-yoon text-md font-medium ${
+        highlight ? '' : 'opacity-60'
```

Comment:
굳이 줄이자면
```suggestion
        !highlight && 'opacity-60'
```
으로도 가능할 것 같긴 합니다ㅏ